src/DEA/allDEA.R

```diff
@@ -1,20 +1,92 @@
+  symbols_real <- symbols[!bool]
+  IDS_real <- other_IDs[!bool]
+  
+  mean_fc <- mean(log2FC_real)
+  stddev <- sd(log2FC_real)
+  p_value <- 2*pnorm(log2FC_real, mean_fc, stddev)
+  
+  p_value_comb <- c(p_value, rep(10^(-50), (length(symbols_pos_inf)+length(symbols_neg_inf))))
+  FC_comb <- c(FC_real, rep(max(FC_real),length(symbols_pos_inf)), rep(min(FC_real),length(symbols_neg_inf)))
+  log2FC_comb <- c(log2FC_real, rep(max(log2FC_real),length(symbols_pos_inf)), rep(min(log2FC_real),length(symbols_neg_inf)))
+  symbols_comb <- c(symbols_real, symbols_pos_inf, symbols_neg_inf)
+  IDS_comb <- c(IDS_real, IDS_pos_inf, IDS_neg_inf)
+  
+  bool_p <- (p_value_comb <= 0.05)
+  
+  non_sig_p_values <- c(rep(max(p_value_comb), length(non_sig_symbols)), p_value_comb[!bool_p])
+  non_sig_symbols <- c(non_sig_symbols, symbols_comb[!bool_p])
+  non_sig_FC <- c(non_sig_FC, FC_comb[!bool_p])
+  non_sig_log2FC <- c(non_sig_log2FC, log2FC_comb[!bool_p])
+  non_sig_IDS <- c(non_sig_IDS, IDS_comb[!bool_p])
+  
+  non_sig_FC[is.nan(non_sig_FC)] = 1
+  non_sig_log2FC[is.nan(non_sig_log2FC)] = 0
+  
+  p_values_sig <- p_value_comb[bool_p]
+  symbols_sig <- symbols_comb[bool_p]
+  FC_sig <- FC_comb[bool_p]
+  log2FC_sig <- log2FC_comb[bool_p]
+  sig_IDS <- IDS_comb[bool_p]
+  
+  if(missing_IDS) {
+    sig_data = data.frame("Symbols" = symbols_sig, "FC" = FC_sig, "log2FC" = log2FC_sig,
+                          "Pvalues" = p_values_sig)
+    
+    non_sig_data = data.frame("Symbols" = non_sig_symbols, "FC" = non_sig_FC, "log2FC" = non_sig_log2FC,
+                              "Pvalues" = non_sig_p_values)
+    
+  } else {
+    sig_data = data.frame("Symbols" = symbols_sig, "FC" = FC_sig, "log2FC" = log2FC_sig,
+                          "Pvalues" = p_values_sig, "otherID" = sig_IDS)
+    
+    non_sig_data = data.frame("Symbols" = non_sig_symbols, "FC" = non_sig_FC, "log2FC", non_sig_log2FC,
+                              "Pvalues" = non_sig_p_values, "otherID" = non_sig_IDS)
+  }
+  
+  return(list(sig_data = sig_data, non_sig_data = non_sig_data))
+  
+}
+
 #### Lyssiotis Imports ####
 
-library(rio)
-# import BxPC3 data as bx
-bx <- import("~jennadiegel/Documents/SURE/DES/Lyssiotis_data/CCLs_BxPC3.csv")
-# import Capan1 data as cp
-cp <- import("~jennadiegel/Documents/SURE/DES/Lyssiotis_data/CCLs_Capan1.csv")
-# import PANC1 data as pn
-pn <- import("~jennadiegel/Documents/SURE/DES/Lyssiotis_data/CCLs_PANC1.csv")
-# import TU8902 data as tu
-tu <- import("~jennadiegel/Documents/SURE/DES/Lyssiotis_data/CCLs_TU8902.csv")
-# import TU8988T data as tut
-tut <- import("~jennadiegel/Documents/SURE/DES/Lyssiotis_data/CCLs_TU8988T.csv")
-# import UM2 data as um2
-um2 <- import("~jennadiegel/Documents/SURE/DES/Lyssiotis_data/CCLs_UM2.csv")
-# import UM90 data as um90
-um90 <- import("~jennadiegel/Documents/SURE/DES/Lyssiotis_data/CCLs_UM90.csv")
+library("rio")
+library("readxl")
+sheets = list('Sheet1','Sheet2','Sheet3','Sheet4','Sheet5','Sheet6','Sheet7')
```

Comment:
Not sure if sheets is used in this script -> could be an artifact from your clean up